src/css/custom.css

```diff
@@ -4,6 +4,11 @@
  * work well for content-centric websites.
  */
 
+@font-face {
+  font-family: 'Manrope';
+  src: url('/font/Manrope-Regular.ttf');
```

Comment:
You need to attribute the author of the font somehow, see the license for this font: https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL 

See the bottom of the link:

"
requirement

The icon shows a person and represents the author(s).
The requirement is for proper attribution of the author(s): name(s) and notice(s) must be preserved and abuse of the name(s) and reputation of the author(s) is forbidden.
See condition 2) and 4) of the OFL."

Either you add it to some license page (please ask Daniel/max) or in the footer

Reply:
@santiagoejm : Did you add the font family licence to a licence page? so we can merge this PR?